save-backend/src/main/kotlin/org/cqfn/save/backend/controllers/DownloadFilesController.kt

```diff
@@ -94,6 +93,29 @@ class DownloadFilesController(
                     ResponseEntity.status(HttpStatus.CONFLICT).build()
                 )
 
+    /**
+     * @param file image to be uploaded
+     * @param owner owner name
+     * @return [Mono] with response
+     */
+    @Suppress("UnsafeCallOnNullableType")
+    @PostMapping(value = ["/api/image/upload"], consumes = [MediaType.MULTIPART_FORM_DATA_VALUE])
+    fun uploadImage(@RequestPart("file") file: Mono<FilePart>, @RequestParam owner: String) =
+            additionalToolsFileSystemRepository.saveImage(file, owner).map { imageInfo ->
+                ResponseEntity.status(
+                    imageInfo.path?.let {
+                        organizationService.saveAvatar(owner, it)
+                        HttpStatus.OK
+                    }
+                        ?: HttpStatus.INTERNAL_SERVER_ERROR
+                )
+                    .body(imageInfo)
+            }
```

Comment:
Fixed

Reply:
Revert it. This throw MissingFieldExeption: Field 'path'. I think it's because of the empty body in Responc
<img width="378" alt="21" src="https://user-images.githubusercontent.com/50615459/155179548-bb3c7366-97a0-4e73-9ee7-be10df93f8cd.PNG">
e

Reply:
You can put `imageInfo` in body, as you did originally. I thought, `imageInfo` is null in this case, but only `imageInfo.path` is null

Reply:
That won't work either. I understand switchIfEmpty is waiting for a ResponseEntity<Unite>, not a ResponseEntity<ImageInfo>